backend/src/test/java/botobo/core/DomainAcceptanceTest.java

```diff
@@ -2,47 +2,38 @@
 
 import botobo.core.admin.dto.AdminCardRequest;
 import botobo.core.admin.dto.AdminWorkbookRequest;
-import botobo.core.auth.AuthAcceptanceTest;
-import io.restassured.RestAssured;
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
-import org.springframework.http.MediaType;
 
 import java.util.List;
 
-public class DomainAcceptanceTest extends AuthAcceptanceTest {
+public class DomainAcceptanceTest extends AcceptanceTest {
 
     public ExtractableResponse<Response> 문제집_생성_요청(AdminWorkbookRequest adminWorkbookRequest) {
-        return RestAssured.given().log().all()
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .auth().oauth2(로그인되어_있음().getAccessToken())
-                .body(adminWorkbookRequest)
-                .when().post("/api/admin/workbooks")
-                .then().log().all()
-                .extract();
+        return request()
+                .post("/api/admin/workbooks", adminWorkbookRequest)
+                .auth()
+                .build()
+                .extractableResponse();
     }
 
-    public void 여러개_문제집_생성_요청(List<AdminWorkbookRequest> requests) {
-        for (AdminWorkbookRequest request : requests) {
-            RestAssured.given().log().all()
-                    .contentType(MediaType.APPLICATION_JSON_VALUE)
-                    .auth().oauth2(로그인되어_있음().getAccessToken())
-                    .body(request)
-                    .when().post("/api/admin/workbooks")
-                    .then().log().all()
-                    .extract();
+    public void 여러개_문제집_생성_요청(List<AdminWorkbookRequest> adminRequests) {
+        for (AdminWorkbookRequest adminRequest : adminRequests) {
+            request()
+                    .post("/api/admin/workbooks", adminRequest)
+                    .auth()
+                    .build()
+                    .extractableResponse();
```

Comment:
```suggestion
        for (AdminWorkbookRequest adminRequest : adminRequests) {
            문제집_생성_요청(adminRequest);
```
별건 아니지만 위에 있는 메서드 활용해서 요렇겐 어떤가요?